toolkit/offchain/src/scripts_data.rs

```diff
@@ -242,13 +242,17 @@ pub(crate) fn reserve_scripts(
 // Returns the simplest MultiSig policy configuration plutus data:
 // there is one required authority and it is the governance authority from sidechain params.
 pub(crate) fn multisig_governance_policy_configuration(
-	governance_authority: MainchainKeyHash,
+	governance_authority: Vec<MainchainKeyHash>,
+	governance_threshold: u8,
 ) -> PlutusData {
 	PlutusData::Array(vec![
-		PlutusData::Array(vec![uplc::PlutusData::BoundedBytes(
-			governance_authority.0.to_vec().into(),
-		)]),
-		PlutusData::BigInt(uplc::BigInt::Int(1.into())),
+		PlutusData::Array(
+			governance_authority
+				.iter()
+				.map(|hash| PlutusData::BoundedBytes(hash.0.to_vec().into()))
+				.collect(),
+		),
+		PlutusData::BigInt(uplc::BigInt::Int((governance_threshold as i64).into())),
```

Comment:
Please avoid as whenever possible, use into or from when additional type annotations are needed
```suggestion
		PlutusData::BigInt(uplc::BigInt::Int((governance_threshold.into()).into())),
```